include/envoy/upstream/cluster_manager.h

```diff
@@ -292,6 +292,15 @@ class ClusterManagerFactory {
                               const absl::optional<envoy::api::v2::core::ConfigSource>& eds_config,
                               ClusterManager& cm) PURE;
 
+  /**
+   * Create an incremental CDS API provider from configuration proto. cds_config should
+   * have api_config_source.api_type == INCREMENTAL_GRPC.
+   */
+  virtual CdsApiPtr
+  createIncrementalCds(const envoy::api::v2::core::ConfigSource& cds_config,
```

Comment:
I don't think this needs to be visible in the `ClusterManager` API; it should be implied by the `ConfigSource` and handled internlly by`createCds`.

Reply:
Done.